docs/data-sources/aws_bucket_policy.md

```diff
@@ -0,0 +1,27 @@
+# databricks_aws_bucket_policy Data Source
+
+This datasource configures simple access policy for AWS S3 buckets, so that Databricks can access data in it. 
+
+## Example Usage
+
+```hcl
+resource "aws_s3_bucket" "this" {
+  bucket = "things"
+  region = "eu-west-1"
+  acl    = "private"
+  force_destroy = true
+}
+
+data "databricks_aws_bucket_policy" "stuff" {
+  bucket_name = aws_s3_bucket.this.bucket
+}
+
+resource "aws_s3_bucket_policy" "this" {
+  bucket     = aws_s3_bucket.this.id
+  policy     = data.databricks_aws_bucket_policy.this.json
+}
+```
+## Argument Reference
```

Comment:
All the other docs make a clear distinction between attribute and argument. 

Argument should be user inputs where attributes are outputs/read only values computed by the resource/datasource. 

The bucket_name should be an argument where the json is an attribute.

This should use the naming convention 
## Argument reference
* `<varname>` - (Required/Computed/Optional) (Datatype) (content)

## Attribute reference
* `<varname>` - (Datatype) (content)